File: cfgs/java_cfgs_foo/NUMBER_DAYS_TANK_WILL_BECOME_EMPTY.java
Nodes
1;special;;;
2;conditional;foo_1 >= foo_2;14;
3;statement;return foo_1;14;
4;exit;;;
5;statement;double eq_root = (fun_1(1 + (8 * (foo_1 - foo_2))) - 1) / 2;15;
6;statement;return ((int) (fun_1(foo_1) + foo_2));16;
Edges
0;2;
1;3;5;
2;4;
3;
4;6;
5;4;

File: cfgs/pythoncfgs/NUMBER_DAYS_TANK_WILL_BECOME_EMPTY.py
Nodes
1;conditional;foo_1 >= foo_2;10;
2;statement;foo_1 = (fun_1(1 + 8 * (foo_2 - foo_3)) - 1) / 2;11;
3;statement;return fun_1(foo_1) + foo_2;12;
4;statement;return foo_1;10;
5;exit;;
Edges
0;2;4;
1;3;
2;5;
3;5;
4;
